go.mod

```diff
@@ -3,34 +3,46 @@ module github.com/kubernetes-sigs/nfs-subdir-external-provisioner
 go 1.14
 
 require (
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/kubernetes v1.18.0
+	github.com/golang/glog v1.0.0
```

Comment:
Just curious - why does the code still use glog?

Reply:
As you noted or mentioned earlier, this repo looks to be unmaintained and afaict, glog usage is side effect of the same. :( This PR bringing the dependency from kube v1.18 ( which is unsupported version) to v1.23 is an attempt to correct to the things.

Reply:
If it's not being maintained, then we should deprecate and archive it.

Reply:
I agree with the sentiment, but I guarantee you people are using this CSI driver and would be upset if it went away. Maybe a call needs to go out putting this project up for adoption.

Reply:
+1 to what @bswartz said. 

Personally am working on making time to maintain this repo. Would love to have some additional help!